runtime/curl.h

```diff
@@ -67,3 +67,29 @@ struct CurlMemoryUsage : vk::not_copyable {
 
   friend class vk::singleton<CurlMemoryUsage>;
 };
+
+class CurlRequest : private vk::not_copyable {
```

Comment:
Let's use only script memory for `CurlRequest`, `CurlResponse` and others if any

Reply:
good suggesstion, done